spec/index.bs

```diff
@@ -4408,6 +4408,7 @@ dictionary GPUTextureViewDescriptor
          : GPUObjectDescriptorBase {
     GPUTextureFormat format;
     GPUTextureViewDimension dimension;
+    GPUTextureUsageFlags usage;
```

Comment:
Note: I imagine for webgpu.h we'll want usage == None to mean inheriting the usages, so we might need something special to say that the usages are exactly 0.

Reply:
Given that creating a view with no usages won't be useful (there's validation to ensure the texture usages aren't 0) is that a scenario we need to support? If you are asking explicitly for a usage of 0 we can pretty confidently say you are doing it wrong, and thus don't need to allow it when 0 can have a reasonable and useful alternative interpretation.

Reply:
Sounds good, would this be a type error or something still surfaced as a validation error on the GPUDevice? (in both cases it is fairly implementable)

Reply:
Maybe I misunderstood, but I thought None would be given the value of 0? In which case the default behavior is to inherit, but you can also specify any combination of other usages (which will be validated against the view format and the texture usages.) The only thing you can't do is specify a view usage of 0, which is OK because that's useless anyway. No new validation needed.

Reply:
Cleared up the confusion offline. I thought the concern was primarily for native interfaces but it's actually about the JS interface where `undefined` and `0` are distinct.

My proposal to handle that is to have the dictionary default to 0 and define that a view usage of 0 means the usage is inherited. That way the native and JS interfaces function more similarly and the JS interface still ends up interpreting `undefined` as "inherit".

The alternative is that we only treat `undefined` as "inherit", throw a validation error if 0 is passed (which matches the texture validation) and native will either differ and treat 0 as inherit or introduce a new, explicit inherit enum. Given that this route involves more opportunity for errors with no real upside AND makes native and JS differ I'm not in favor of it.

Reply:
WG approved of defaulting to 0 and having that mean inherit.